### ISSUE TITLE
Global traffic

### ISSUE THREAD

**Issue by DrPandemic** [erlanglab/erlangpl#48]
I'm really unsure where this question should be asked, so I'm posting it here.

On the traffic page, is it possible to see the communication between all nodes? If I'm not mistaken, currently we can only see messages between the inspected nodes and the others.

If it's not currently possible, would it necessitate a lot of modifications in the erlangpl core to achieve this?

**Comment by baransu** [comment]
Right now it's not possible.
From what I understand it would cost a lot to get all traffic in cluster but I don't really know if and how we want to support this.

/cc @michalslaski @arkgil

**Comment by arkgil** [comment]
Yes, it is not possible given our current approach to monitoring. Right now we spawn only one process on monitored node which provides us all information needed to render the views.

This approach have its limitations, one of them is not being able to track the traffic between all the nodes in the cluster - to get this data we use `net_kernel:nodes_info/0` function which only returns amount of data going in and out of the node on which it was called.

**Comment by michalslaski** [comment]
@DrPandemic if you could help us by answering some of the below questions, we can try to design a view specific to your needs.

1. how many nodes do you have in your cluster?

2. are you interested only in visualizing inter-node traffic or do you need to see message passing between processes too?

3. Do you use any specific product like Phoenix, Riak, RabbitMQ, MongooseIM, VerneMQ, etc. ?

**Comment by DrPandemic** [comment]
Hey people, thank you for your time.

1. I'm aiming to visualize a cluster of maybe 5 to 10 nodes. What would be the price for each new node?

2. I was hoping for inter-node traffic. Only being able to see messages between the observed node's processes is enough.

3. My current project is a cluster of elixir apps. I was considering trying to connect a node with Phoenix, but not in the immediate future.
